Documentation/user-manual.txt

```diff
@@ -1513,7 +1538,7 @@ assumes there is an Internet connection each time this item is viewed within
 _Subsurface_.
 
 Having selected the local folder or Internet image to be imported, the time
-synchronization dialog appears (see image below). The time
+synchronization dialogue appears (see image below). The time
```

Comment:
ok, I'll stop commenting on 'dialog' - let's please not change that word.

Reply:
done